js/app/restaurant/components/Order/ChangeRestaurantOnEditFulfilmentDetailsModal.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import Modal from 'react-modal'
+import { useTranslation } from 'react-i18next'
+
+export default function ChangeRestaurantOnEditFulfilmentDetailsModal({
```

Comment:
Small typo, should be `ChangeRestaurantOnEditFulfillmentDetailsModal` (missing L)

Reply:
In the filename as well

Reply:
It's British English 😄 https://dictionary.cambridge.org/uk/dictionary/english/fulfilment